### DISCUSSION TITLE
Comments API

### DISCUSSION THREAD

**Discussion by lemkek** [GitbookIO/community#487] [Feature Requests]
Create a comment API to add and reply to comments on a given space for a given user. It would be nice to resolve comments previously made as well via the api. We would like to use this api to interact with our Transportation Management System.

**Comment by SamyPesse** [comment]
There is already an API you can use to do everything the app is doing on comments. It's not documented yet, but you can find references to it in https://api.gitbook.com/openapi.yaml

If you are using JS/Typescript, you can also use our API client (https://www.npmjs.com/package/@gitbook/api), and all operations will be fully typed.

**Reply by lemkek** [reply]
perfect thanks!
